x-pack/filebeat/modules.d/googlecloud.yml.disabled

```diff
@@ -19,3 +19,21 @@
     # Credentials file for the service account with authorization to read from
     # the subscription.
     var.credentials_file: ${path.config}/gcp-service-account-xyz.json
+
+  firewall:
+    enabled: true
+
+    # Google Cloud project ID.
+    var.project_id: my-gcp-project-id
+
+    # Google Pub/Sub topic containing firewall logs. Stackdriver must be
+    # configured to use this topic as a sink for firewall logs.
+    var.topic: googlecloud-vpc-firewall
+
+    # Google Pub/Sub subscription for the topic. Filebeat will create this
+    # subscription if it does not exist.
+    var.subscription_name: filebeat-googlecloud-firewall-sub
+
+    # Credentials file for the service account with authorization to read from
+    # the subscription.
+    var.credentials_file: ${path.config}/gcp-service-account-xyz.json
```

Comment:
Not related? CI jobs seem to be failing because of this. Run `mage fmt update` and commit the changes.

Reply:
Yes, this was removed accidentally on https://github.com/elastic/beats/pull/14553. We want it back though. Will run the updates and push again, thanks!

Reply:
Yes the whole file was removed on https://github.com/elastic/beats/pull/14553 accidentally. Will run the updates, thank you!